stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/plugin/logic/channel/internal/ChannelLogic.kt

```diff
@@ -410,15 +410,16 @@ internal class ChannelLogic(
      * @param message [Message].
      */
     private fun incrementUnreadCountIfNecessary(message: Message) {
-        val currentUserId = globalMutableState.user.value?.id ?: return
+        val user = globalMutableState.user.value ?: return
+        val currentUserId = user.id ?: return
```

Comment:
This return is redundant
```suggestion
        val currentUserId = user.id
```